muspinsim_input/muspinsim_input.xml

```diff
@@ -180,72 +192,87 @@
                                 Euler angles helper function: eulrange(n).
                                 Zaremba-Conroy-Wolfsberg helper function: zcw(n)">
                         <option value="2_polar">Polar angles</option>
                         <option value="3_euler">Euler angles</option>
-                        <option value="4_euler">Euler angles with 1 weight</option>
+                        <option value="4_euler">Euler angles with weight</option>
                         <option value="eulrange">Euler angles helper function (eulrange(n))</option>
-                        <option value="zcw">Zaremba-Conroy-Wolfsberg helper function (zcw(n))</option>
+                        <option selected="true" value="zcw">Zaremba-Conroy-Wolfsberg helper function (zcw(n))</option>
                     </param>
                     <when value="eulrange">
-                        <param name="eul_n" type="text" value="0" label="value for n for eulrange(n)"
-                               help="Enter number (larger numbers are more computationally expensive).
-                               Allows default expressions, constants and functions"/>
+                        <param name="eul_n" type="text" value="0" label="N"
+                               help="value for n for eulrange(n),
+                               Allows default expressions, constants and functions
+                               WARNING: large values are more computationally expensive."
+                        />
                     </when>
                     <when value="zcw">
-                        <param name="zcw_n" type="text" value="0" label="value for n for zcw(n)"
-                               help="Enter number (larger numbers are more computationally expensive).
-                               09Allows default expressions, constants and functions"/>
-
+                        <param name="zcw_n" type="text" value="0" label="N"
+                               help="value for n for zcw(n)
+                               Allows default expressions, constants and functions
+                               WARNING: large values are more computationally expensive."
+                        />
                     </when>
                     <when value="2_polar">
-                        <param name="theta" type="text" value="" label="theta polar angle"
-                               help="Allows default expressions, constants and functions"/>
-                        <param name="phi" type="text" value="" label="phi polar angle"/>
+                        <param name="theta" type="text" value="" label="θ (theta) angle"
+                               help="All polar angle entries allow default expressions, constants and functions"/>
+                        <param name="phi" type="text" value="" label="ϕ (phi) angle"
+                               help="All polar angle entries allow default expressions, constants and functions"/>
```

Comment:
One of these is the 'azimuth' and one is the 'inclination' - but apparently [there are competing conventions for which is which](https://en.wikipedia.org/wiki/Spherical_coordinate_system#Conventions)... I assume muspinsim follows the ISO definition where theta=inclination and phi=azimuth. I think these terms should be included somewhere here.